ospkg/zip.go

```diff
@@ -62,5 +57,5 @@ func unzipFile(archive *zip.Reader, name string) ([]byte, error) {
 
 	stlog.Debug("cannot find %s in archive", name)
 
-	return nil, ErrUnzip
+	return nil, ErrFailedToUnzip
```

Comment:
this would cause a duplicate ErrFailedToUnzip Text output in `ospkg.unzip()`, since error.Error() == Info.
It should probably output "ospkg not found" os something similar.